### DISCUSSION TITLE
Support domain connection

### DISCUSSION THREAD

**Discussion by Jengro777** [rustdesk/rustdesk#5432] [Feature Request]
At present, it can only be connected through iD or ipv4 address, hoping to support the domain name after ipv4 and ipv6 resolution for connection

**Comment by dinger1986** [comment]
Really? Mine connects fine through domain

**Reply by Jengro777** [reply]
1. The IPv4 domain name has not been brought into port 21118, and this issue is no longer present

2. The ipv6 domain name is still unavailable

**Reply by Jengro777** [reply]
The ipv6 domain name may be a problem with remote mac wining
